### PR TITLE
markdig fixes

### LINKED ISSUE #7201: Markdig migration preparation - master issue - round 2
The docs below, in the *master* branch, were flagged during the test run of the Markdig migration. Refer to the spreadsheet's *DifferenceAfterMigration* tab for more details, and apply the following column filter:

- *Migrated* = True

The *SourceMarkdown* column contains the offending Markdown. Changes can be validated in the *markdig-migration* branch. Refer to [Markdig Migration Information](https://review.docs.microsoft.com/en-us/new-hope/engineering/tools/markdig/markdig-migration-information?branch=master&tabs=dfm) for syntax guidance.

## _vblang
- [ ] spec/general-concepts.md

## docs/core
- [x] whats-new/dotnet-core-2-1.md

## docs/framework
- [x] whats-new/index.md
- [x] network-programming/certificate-selection-and-validation.md
- [x] network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md
- [x] wpf/advanced/use-automatic-layout-overview.md

## docs/fsharp
- [x] tutorials/type-providers/creating-a-type-provider.md

## docs/standard
- [x] io/how-to-find-existing-files-and-directories-in-isolated-storage.md
- [x] base-types/custom-date-and-time-format-strings.md
- [x] base-types/custom-timespan-format-strings.md
- [x] base-types/enumeration-format-strings.md

## docs/visual-basic
- [x] language-reference/queries/aggregate-clause.md
- [x] programming-guide/language-features/linq/introduction-to-linq.md

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Markdig changes.

Fixes #7201

### CODE REVIEW

File: docs/framework/network-programming/certificate-selection-and-validation.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The <xref:System.Net> classes support several ways to select and validate <xref:
 ## Tools for Certificate Configuration  
  A number of tools are available for client and server certificate configuration.  
   
- The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at www.microsoft.com.  
+ The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at [www.microsoft.com]().
```
Comment (scottaddie) [inline docs/framework/network-programming/certificate-selection-and-validation.md]: The Microsoft URL is missing inside the parens.

**Review by scottaddie — CHANGES_REQUESTED** [review]

File: docs/framework/network-programming/certificate-selection-and-validation.md (hunk 1 above)
Comment (Thraka) [reply docs/framework/network-programming/certificate-selection-and-validation.md]: Whoops I mean to do `<www.microsoft.com>`

Comment (Thraka) [reply docs/framework/network-programming/certificate-selection-and-validation.md]: Actually no I was correct 😄  the commonmark spec says it's OK: https://spec.commonmark.org/0.28/#example-461

Comment (scottaddie) [reply docs/framework/network-programming/certificate-selection-and-validation.md]: It looks like the resulting anchor tag's `href` attribute will be empty if you stick with what you have now.

Comment (Thraka) [reply docs/framework/network-programming/certificate-selection-and-validation.md]: ugh you're right. I'm stupid :) I'll fix it

File: docs/framework/network-programming/certificate-selection-and-validation.md [hunk 2]
```diff
@@ -35,13 +35,13 @@ The <xref:System.Net> classes support several ways to select and validate <xref:
 ## Tools for Certificate Configuration  
  A number of tools are available for client and server certificate configuration.  
   
- The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at www.microsoft.com.  
+ The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at [www.microsoft.com](http://www.microsoft.com).
```
Comment (scottaddie) [inline docs/framework/network-programming/certificate-selection-and-validation.md]: http --> https

File: docs/framework/network-programming/certificate-selection-and-validation.md [hunk 3]
```diff
@@ -35,13 +35,13 @@ The <xref:System.Net> classes support several ways to select and validate <xref:
...
+ The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at [www.microsoft.com](http://www.microsoft.com).  
   
 The *HttpCfg.exe* tool can be used to configure server certificates for the <xref:System.Net.HttpListener> class. The *HttpCfg.exe* tool is provided as one of the support tools for Windows Server 2003 and Windows XP Service Pack 2. *HttpCfg.exe* and the other support tools are not installed by default on either Windows Server 2003 or Windows XP. On Windows Server 2003. the support tools are installed separately from the following folder and file on the Windows Server 2003 CD-ROM:  
   
  \Support\Tools\Suptools.msi  
   
- For use with Windows XP Service Pack 2, the Windows XP Support Tools are available as a download from www.microsoft.com.  
+ For use with Windows XP Service Pack 2, the Windows XP Support Tools are available as a download from [www.microsoft.com](http://www.microsoft.com).
```
Comment (scottaddie) [inline docs/framework/network-programming/certificate-selection-and-validation.md]: http --> https

File: docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md [hunk 4]
```diff
@@ -50,9 +50,9 @@ Several changes were made to the <xref:System.Uri?displayProperty=nameWithType>
   
 -   <xref:System.Uri.IsLoopback%2A> now produces consistent results.  
   
--   The URI "`file:///path`" is no longer translated into "file://path".  
+-   The URI "`file:///path`" is no longer translated into `file://path`.
```
Comment (mairaw) [inline docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md]: should you keep the quotes as the first one also has them? or remove from both?

File: docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md [hunk 5]
```diff
@@ -50,9 +50,9 @@ Several changes were made to the <xref:System.Uri?displayProperty=nameWithType>
   
 -   <xref:System.Uri.IsLoopback%2A> now produces consistent results.  
   
--   The URI "`file:///path`" is no longer translated into "file://path".  
+-   The URI "`file:///path`" is no longer translated into `file://path`.  
   
--   "#" is now recognized as a host name terminator. That is, "http://consoto.com#fragment" is now converted to "http://contoso.com/#fragment".  
+-   "#" is now recognized as a host name terminator. That is, `http://consoto.com#fragment` is now converted to `http://contoso.com/#fragment`.
```
Comment (mairaw) [inline docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md]: should the # here be escaped `\#`

File: docs/framework/wpf/advanced/use-automatic-layout-overview.md [hunk 6]
```diff
@@ -44,11 +44,11 @@ Auto Resizable Button
...
-| Do not set a fixed size for a window. | -   Use <xref:System.Windows.Window.SizeToContent%2A>.<br />-   For example:<br /><br /> [!code-xaml[LocalizationGrid#2](../../../../samples/snippets/csharp/VS_Snippets_Wpf/LocalizationGrid/CS/Pane1.xaml#2)] |
+| Do not use absolute positions. | <ul><li>Do not use <xref:System.Windows.Controls.Canvas> because it positions elements absolutely.</li><li>Use <xref:System.Windows.Controls.DockPanel>, <xref:System.Windows.Controls.StackPanel>, and <xref:System.Windows.Controls.Grid> to position controls.</li><li>For a discussion about various types of panels, see [Panels Overview](../../../../docs/framework/wpf/controls/panels-overview.md).</li></ul> |
+| Do not set a fixed size for a window. | -   Use <xref:System.Windows.Window.SizeToContent%2A>.<br />For example:<br /><br /> [!code-xaml[LocalizationGrid#2](../../../../samples/snippets/csharp/VS_Snippets_Wpf/LocalizationGrid/CS/Pane1.xaml#2)] |
```
Comment (mairaw) [inline docs/framework/wpf/advanced/use-automatic-layout-overview.md]: use `<li>` here too or remove -? 

image

Comment (Thraka) [reply docs/framework/wpf/advanced/use-automatic-layout-overview.md]: There is a problem with the - lists. It seems to pick up the first - but the br tags that would normally break things up and start a new bullet doesn't seem to work. So i just forced the bullets.

File: docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md (hunk 5 above)
Comment (Thraka) [reply docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md]: I dont think # needs to be escaped. # only would need escape if it was at the start of a line.

File: docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md (hunk 4 above)
Comment (Thraka) [reply docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md]: Either way. I just tried to fix the issue, not redesign it 😄

**Comment by Thraka** [comment]
@scottaddie fixed the issues you mentioned :smile:

**Review by scottaddie — APPROVED** [review]

**Pull request merged**